[2024-12-03T16:01:27.711+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_file_processing.etl_merge manual__2024-12-03T16:01:05.751106+00:00 [queued]>
[2024-12-03T16:01:27.719+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_file_processing.etl_merge manual__2024-12-03T16:01:05.751106+00:00 [queued]>
[2024-12-03T16:01:27.720+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-12-03T16:01:27.731+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): etl_merge> on 2024-12-03 16:01:05.751106+00:00
[2024-12-03T16:01:27.742+0000] {standard_task_runner.py:57} INFO - Started process 123 to run task
[2024-12-03T16:01:27.746+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_file_processing', 'etl_merge', 'manual__2024-12-03T16:01:05.751106+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/ETL_file_processing.py', '--cfg-path', '/tmp/tmptzjtpmza']
[2024-12-03T16:01:27.747+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask etl_merge
[2024-12-03T16:01:27.789+0000] {task_command.py:410} INFO - Running <TaskInstance: etl_file_processing.etl_merge manual__2024-12-03T16:01:05.751106+00:00 [running]> on host effe9442f8f2
[2024-12-03T16:01:27.854+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_file_processing' AIRFLOW_CTX_TASK_ID='etl_merge' AIRFLOW_CTX_EXECUTION_DATE='2024-12-03T16:01:05.751106+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-03T16:01:05.751106+00:00'
[2024-12-03T16:01:30.784+0000] {logging_mixin.py:150} INFO - No data to merge.
[2024-12-03T16:01:30.788+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-12-03T16:01:30.857+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=etl_file_processing, task_id=etl_merge, execution_date=20241203T160105, start_date=20241203T160127, end_date=20241203T160130
[2024-12-03T16:01:31.015+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-12-03T16:01:31.180+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
